src/model.jl

```diff
@@ -279,6 +279,28 @@ function create_model(
         set_lower_bound(storage_level[a, rp, P[(a, rp)][end]], graph[a].initial_storage_level)
     end
 
+    # Investment limits
+    for a ∈ Ai
+        if graph[a].capacity > 0 && !ismissing(graph[a].investment_limit)
+            set_upper_bound(assets_investment[a], graph[a].investment_limit / graph[a].capacity)
+        end
+    end
+
+    @expression(
+        model,
+        flow_max_capacity[(u, v) ∈ F],
+        max(graph[u, v].export_capacity, graph[u, v].import_capacity)
+    )
+
+    for (u, v) ∈ Fi
+        if flow_max_capacity[(u, v)] > 0 && !ismissing(graph[u, v].investment_limit)
+            set_upper_bound(
+                flows_investment[(u, v)],
+                graph[u, v].investment_limit / flow_max_capacity[(u, v)],
+            )
+        end
+    end
```

Comment:
Since the import and export capacities are only numbers, using expressions is overkill.
I just them explicitly below (the linter might complain). If you want to keep the variables for future use, then maybe it makes sense to create a different structure. If you just want to write a shorter equation, we can create the variable inside the for.
```suggestion
    for (u, v) ∈ Fi
        if flow_max_capacity[(u, v)] > 0 && !ismissing(graph[u, v].investment_limit)
            set_upper_bound(
                flows_investment[(u, v)],
                graph[u, v].investment_limit / max(graph[u, v].export_capacity, graph[u, v].import_capacity),
            )
        end
    end
```